README.md

```diff
@@ -71,7 +82,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+I believe the code would have to be placed into the model. I say this because your controller should by as DRY as possible, only with the necessary RESTful attributes. A helper method is written for the view but from my understanding, any method that will be long and deals with the behavior of an object should go in the model. You would right a method title ratings and define how you want that method to behave in the inside. You can call this method anywhere within the application without making the controller or view cluttered. Keeping the controller and view files clean and easier for the router to read.
 ```
```

Comment:
:+1: you're right on almost all of this! the question is technically dealing with manipulating html, in which case we'd use a helper. but your definitions are correct!